### ISSUE TITLE
specify write location

### ISSUE THREAD

**Issue by presentoccupant** [hawkfish/olapscript#79]
Is there a way to specify where in a sheet a particular block of data (anything from a cell, to a row, to a column, to a 2-d range) gets written?  It looks like .tosheet always starts at A1.

I guess this is another way of asking if there's the equivalent of setRow(s) and setColumn(s) methods.

**Comment by hawkfish** [comment]
No, but that was in my radar. Now it is written down so it doesn't get lost!

**Comment by hawkfish** [comment]
One interesting wrinkle is clearing the sheet. maybe that needs to be an option?

**Comment by presentoccupant** [comment]
Yes, I think it does need to be an option.  It would also be nice to just be able to say "start at the next blank row," although if doing the calculation ahead of time and passing that information in is preferable, that's fine too.

**Comment by presentoccupant** [comment]
...or alternatively, provide a row and column index as the upper left corner of the range to be written.
